src/app/search-bar/search-bar.component.ts

```diff
@@ -113,6 +115,9 @@ export class SearchBarComponent implements OnInit, AfterViewInit {
 
   ngOnInit() {
     this.displayStatus = 'hidebox';
+    this.boxWidth = document.getElementById('nav-group').style.width.replace('px', '');
```

Comment:
No, @nikhilrayaprolu we cannot, we need different values for index and results page. From css, we can set only one value.